modules/javafx.controls/src/test/java/test/javafx/scene/control/PasswordFieldTest.java

```diff
@@ -94,4 +97,21 @@ public void setup() {
         pwdField.setText(string);
         assertEquals(string.length()-1, pwdField.getLength());
     }
+
+    @Test public void accessibleAttributeText() {
```

Comment:
Thank you for adding the test!

we usually place `@Test` on a separate line (I know you followed the existing style).

also, please update the copyright year to 2025.

otherwise, looks good.